SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -111,7 +111,7 @@ void CFlowOutput::AddAnalyzeSurfaceOutput(CConfig *config){
 void CFlowOutput::SetAnalyzeSurface(CSolver *solver, CGeometry *geometry, CConfig *config, bool output){
 
   unsigned short iDim, iMarker, iMarker_Analyze;
-  unsigned long iVertex, iPoint;
+  unsigned long iVertex, iPoint, iVertex_Normal, iPoint_Normal;
```

Comment:
Don't declare more variables here, we do it everywhere but it is not the c++ way and easily leads to bugs.